src/images/badges/phenominal_woman.svg

```diff
@@ -0,0 +1,30 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="255.167" height="35.059" viewBox="0 0 255.167 35.059">
```

Comment:
Thank you for the PR!

It seems like the SVG didn't update with your changes as it's rendering the `You Didn't Ask For This` badge.
![image](https://user-images.githubusercontent.com/8878152/92512483-d82abc80-f1dc-11ea-8a74-495003fbe9cd.png).